ui/apps/dashboard/e2e/policy/overridepolicy/test-utils.ts

```diff
@@ -0,0 +1,425 @@
+/*
+Copyright 2025 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Page } from '@playwright/test';
+import * as k8s from '@kubernetes/client-node';
+import { V1ObjectMeta } from '@kubernetes/client-node';
+import { parse } from 'yaml';
+import { expect } from '@playwright/test';
+import {
+    setupDashboardAuthentication,
+    setMonacoEditorContent,
+    debugScreenshot,
+    waitForResourceInList,
+    getResourceNameFromYaml,
+    createKarmadaApiClient
+} from '../../test-utils';
+import type { DeepRequired } from '../../test-utils';
+
+// Re-export ALL utilities from parent test-utils for single-point import
+export {
+    setupDashboardAuthentication,
+    setMonacoEditorContent,
+    debugScreenshot,
+    waitForResourceInList,
+    getResourceNameFromYaml,
+    createKarmadaApiClient
+};
+
+// Re-export types separately
+export type { DeepRequired };
+
+// Override Resource Configurations
+export const RESOURCE_CONFIGS = {
+    overridepolicy: {
+        kind: 'OverridePolicy',
+        apiGroup: 'policy.karmada.io',
+        apiVersion: 'v1alpha1',
+        plural: 'overridepolicies',
+    },
+    clusteroverridepolicy: {
+        kind: 'ClusterOverridePolicy',
+        apiGroup: 'policy.karmada.io',
+        apiVersion: 'v1alpha1',
+        plural: 'clusteroverridepolicies',
+    }
+} as const;
+
+/**
+ * Generic function to create Karmada OverridePolicy resources
+ */
+export async function createK8sResource<ResourceType extends keyof typeof RESOURCE_CONFIGS>(
```

Comment:
Is this function name not quite appropriate? OverridePolicy should not be a Kubernetes resource, right?